identity/src/ecdsa.rs

```diff
@@ -102,17 +109,28 @@ impl SecretKey {
         signature.as_bytes().to_owned()
     }
 
-    /// Encode a secret key into a byte buffer.
+    /// Convert a secret key into a byte buffer.
     pub fn to_bytes(&self) -> Vec<u8> {
         self.0.to_bytes().to_vec()
     }
 
     /// Decode a secret key from a byte buffer.
-    pub fn from_bytes(buf: &[u8]) -> Result<Self, DecodingError> {
+    #[deprecated(
+        since = "0.2.0",
+        note = "This method name is inaccurate, use `SecretKey::try_from_bytes` instead"
```

Comment:
What do you think of saying instead:

```suggestion
        note = "This method name does not follow Rust conventions, use `SecretKey::try_from_bytes` instead"
```

Reply:
Please apply the same to the others too.